Review IGs 202212.0 (#1591)
spryker/spryker-docs: _includes/pbc/all/install-features/202212.0/install-the-availability-notification-feature.md

{% info_block warningBox “Verification” %}

We recommend setting `AVAILABILITY_NOTIFICATION_CHECK_PRODUCT_EXISTS` to true. 

> Make sure that don't catch the previously mentioned exception somewhere in your Pyz code but use the check of `$availabilityNotificationSubscriptionResponseTransfer->getIsSuccess()`.

Reviewer: Double-check this sentence

Author: Rewritten
```suggestion
Make sure that the previously mentioned exception is not thrown somewhere in your Pyz code; however, use the check of `$availabilityNotificationSubscriptionResponseTransfer->getIsSuccess()`.
```
